### PR TITLE
Lr/restart tests

### PR DESCRIPTION
**Pull request by lukaszrzasik**
Closes #<ISSUE_NUMBER>

### CODE REVIEW

File: .github/workflows/slowtest.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ env:
   RUST_LOG: info,libp2p=off,node=error
 
 jobs:
-  slow-tests:
+  slowest:
```
Comment (tbro) [inline .github/workflows/slowtest.yaml:25]: I think some unwanted change may have been pulled in here.

Comment (jbearer) [reply .github/workflows/slowtest.yaml:25]: Seems likely, I can revert this

**Comment by tbro** [comment]
How is this test intended to be run? It is not linked into the module tree so things like `cargo test` will not find it.

**Review by tbro — APPROVED** [review]
There's a lot in there. Looks good as far as I can tell.

**Comment by jbearer** [comment]
CI here is no more broken than on main, and I checked that the restart tests themselves are passing, so I'm gonna merge this

**Pull request merged**